[te] Benchmark comparing fused overhead to unfused

bertmaher opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #50318 [te] Optimize allocation of kernel outputs
* **#50305 [te] Benchmark comparing fused overhead to unfused**

That's it

Differential Revision: [D25856891](https://our.internmc.facebook.com/intern/diff/D25856891/)

bertmaher committed:
[te] Benchmark comparing fused overhead to unfused

That's it

Differential Revision: [D25856891](https://our.internmc.facebook.com/intern/diff/D25856891/)

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
👍

ZolotukhinM reviewed (commented):
One question: shouldn't the new file be added to `benchmarks/cpp/tensorexpr/CMakeLists.txt`?

bertmaher commented:
> One question: shouldn't the new file be added to `benchmarks/cpp/tensorexpr/CMakeLists.txt`?

Oh yes good call.  I've been working too much in fbcode lately :)

bertmaher committed:
add to cmakelists on "[te] Benchmark comparing fused overhead to unfused"


That's it

Differential Revision: [D25856891](https://our.internmc.facebook.com/intern/diff/D25856891/)

[ghstack-poisoned]

ZolotukhinM commented:
> > One question: shouldn't the new file be added to `benchmarks/cpp/tensorexpr/CMakeLists.txt`?
> 
> Oh yes good call. I've been working too much in fbcode lately :)

It seems that `bench_approx.cpp` is already missing there btw :)
